src/sqlfluff/cli/commands.py

```diff
@@ -678,11 +716,13 @@ def fix(
             )
             click.echo(
                 colorize(
-                    "Use '--ignore templating' to attempt to fix anyway.",
+                    "Use '--ignore templating' and '--fix-even-unparsable' to "
+                    "attempt to fix anyway.",
```

Comment:
Do you need both? I thought you only needed the latter and the former just controlled the exit code? Though not sure how to succinctly put that!

What about:

```suggestion
                    "Use --fix-even-unparsable' to attempt to fix the SQL anyway, ",
                    " and use '--ignore templating' to avoid outputing templating ",
                    " errors as well.",
```

Reply:
Will do.

Reply:
Technically, you're absolutely right that only `--fix-even-unparsable` is now needed. I was unsure of the best wording here, and thought that users may want to be aware of both options. It's a good point, though. I'll remove mention of `--ignore templating`.

Reply:
Done.